src/libraries/Microsoft.Extensions.Primitives/src/StringValues.cs

```diff
@@ -740,12 +740,12 @@ public override int GetHashCode()
                 {
                     return Unsafe.As<string>(this[0])?.GetHashCode() ?? Count.GetHashCode();
                 }
-                var hcc = default(HashCodeCombiner);
+                int hashCode = 0;
```

Comment:
How do we confirm the two logics yield the same results? Was going to recommend we move the two removed tests to where HashHelpers lives but could not find where `src\libraries\System.Private.CoreLib\src\System\Numerics\Hashing\HashHelpers.cs` tests are placed.

cc: @tannergooding

Reply:
I don't know if we had any tests for it, it was an internal helper type.

Reply:
> How do we confirm the two logics yield the same results?

GetHashCode doesn't need to return the exact same results across versions. It just needs to return the same results for 2 logically equivalent objects.